routes/api/users.js

```diff
@@ -27,6 +30,20 @@ router.get('/', async (req,res)=>{
     });
 });
 
+//endpoint para buscar user por su id
+router.get('/:userId',isAdmin, async (req,res)=>{
+    if (isNaN(req.params.userId)) {

```

Comment:
```suggestion
    if (Number.isNaN(req.params.userId)) {
```
Lo mismo que en stickers y eventos